specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Storage/Storage.tsp

```diff
@@ -200,6 +200,12 @@ namespace Microsoft.EventGrid.SystemEvents {
     /** The type of blob. */
     blobType?: string;
 
+    /** The current tier of the blob. */
+    accessTier?: string;
+
+    /** The previous tier of the blob. */
+    previousTier?: string;
```

Comment:
Can you please generate the swagger for these TypeSpec events as well, see [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/README.md#for-service-system-events-pr-approval)

Reply:
Updated the PR with the generated swagger

Reply:
Please also update the example files with these properties.

Reply:
Updated the example files